src/workloads/replication/startup/1_0_5GB.yml

```diff
@@ -0,0 +1,62 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: >
+  Loads the data for the light phase.
+  To know more about the test phases please refer to 'src/workloads/replication/startup/README.md'.
+
+  Expected behavior:
+  --------------
+  We expect after restarting mongod after running this workload to not having any ops to be applied
+  during startup recovery:
+
+  Sample logs:
+  +-------------------------------------------------------------------------------------------+
+  | {                                                                                         |
+  |     s: "I",                                                                               |
+  |     c: "REPL",                                                                            |
+  |     id: 21549,                                                                            |
+  |     ctx: "initandlisten",                                                                 |
+  |     msg: "No oplog entries to apply for recovery. Start point is at the top of the oplog" |
+  |   }                                                                                       |
+  +-------------------------------------------------------------------------------------------+
+
+Keywords:
+- startup
+- collections
+- indexes
+- defaultWC
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2500
+
+NumPhases: &max_phase 1
```

Comment:
nit: could we add these params into a named block?

That is (something like):

```
GlobalDefaults:
  - NumPhases: &max_phase 1
  - Database: &database "startup_5GB"
  - CollectionCount: &collectionCount 5
```

Functionally this does nothing but it does at least show that these are a set of the params.

Reply:
Done